src/shared/menu.ts

```diff
@@ -278,8 +278,10 @@ export function makeMenuDropdown(
     active?: (state: EditorState) => boolean,
     ...children: MenuCommandEntry[]
 ): MenuCommandEntry {
-    const popoverId = `${key}-popover`;
-    const buttonId = `${key}-btn`;
+    const randomId = generateRandomId();
+    const popoverId = `${key}-popover-${randomId}`;
```

Comment:
I made a change here to automatically add an id to all menu dropdowns automatically so consumers don't have to remember to do it themselves.